### DISCUSSION TITLE
Fuel Mix Endpoint

### DISCUSSION THREAD

**Discussion by fredonia88** [ercot/api-specs#41] [Q&A]
After surfing [the docs](https://developer.ercot.com/applications/pubapi/user-guide/openapi-documentation/), I have a couple questions: 

1. Is there any available endpoint that serves fuel mix data? i.e. solar vs wind vs natural gas? Such as [here](https://www.ercot.com/gridmktinfo/dashboards/fuelmix)
2. Which endpoints would I use to derive the numbers shown in Ercot's [Supply and Demand summary](https://www.ercot.com/gridmktinfo/dashboards/supplyanddemand)? i.e. demand, committed capacity, demand forecast, etc.

**Comment by mackermann-ercot** [comment]
Hello fredonia88,

Currently, we are serving averaged actual and forecasted values (5 minute and hourly) for both Wind and Solar, however we do not yet have the same for other generation sources at this time.  I will pass along your request to one of our Business Information analysts and determine if/when we will be serving that data via API.

Regards,
-mackermann, ERCOT

**Comment by FelixDQ** [comment]
Just adding that we at https://app.electricitymaps.com/ would be very interested in this endpoint as well. We are currently showing data from EIA, but it would be great if we could show real-time data from the ERCOT API instead.

**Reply by mackermann-ercot** [reply]
Thank you for the input Felix.  I will pass your interest along as well.
